packages/smart-contracts/packages/staking/contracts/Staking.sol

```diff
@@ -283,11 +283,12 @@ contract Staking is
         uint32 distributionPercentage
     ) external onlyRole(DEFAULT_ADMIN_ROLE) mPoolExists(poolId) {
         // Checks: distribution percentage should not exceed 100%
-        if (distributionPercentage > uint48(PERCENTAGE_PRECISION)) {
+        if (distributionPercentage > PERCENTAGE_PRECISION) {

```

Comment:
It's okay, but I'm not sure why there was an `if (distributionPercentage > uint48(PERCENTAGE_PRECISION))` check for `uint48` before

Reply:
`PERCENTAGE_PRECISION` was converted to `uint32`, but not `distributionPercentage`